### PR TITLE
Update Research Page

### PR DESCRIPTION
**Pull request by kirubarajan**
Implemented: 
- naming changes
- meta-data for cards
- infinite scroll for companies

### CODE REVIEW

**Comment by kirubarajan** [comment]
Current Screenshot:

Screenshot

Currently implementing infinite scroll.

**Comment by kirubarajan** [comment]
Current `PAGE_LENGTH` for buffering infinite scroll parameter is set to 2, any ideas on what to set it to?

**Comment by kirubarajan** [comment]
Something to note is that I updated the type signature of `retrieveCompanies(limit: number, skip: number)` so that it accepts parameters to assist with pagination. I checked the codebase and the only reference to this function is in the `Archive.jsx` component anyway.

**Comment by kirubarajan** [comment]
Also should I rename the `Archive` components to `Research` as well?

**Comment by VinaiRachakonda** [comment]
Yea down to rename to research

**Comment by VinaiRachakonda** [comment]
> Current `PAGE_LENGTH` for buffering infinite scroll parameter is set to 2, any ideas on what to set it to?

What does this parameter do?

File: client/components/Archive/Archive.tsx [hunk 1]
```diff
@@ -38,6 +48,27 @@ export default class Archive extends Component<{}, ArchiveState> {
     }
   }
 
+  // helper function which buffers the current state's list
+  // state list is displayed in InifiniteScroll component
+  loadMoreItems = (ac: ArchiveContainer, cuc: CurrentUserContainer) => {
+    ac.retrieveCompanies(PAGE_LENGTH, this.state.skip).then((pagination) => {
```
Comment (VinaiRachakonda) [inline client/components/Archive/Archive.tsx]: Will these retrieve repeats?

**Comment by kirubarajan** [comment]
`PAGE_LENGTH` is the amount of companies to retrieve at a time.

File: client/components/Archive/Archive.tsx (hunk 1 above)
Comment (kirubarajan) [reply client/components/Archive/Archive.tsx]: The combination of the number of items to skip and the number of items to retrieve make sure we don't retrieve the same company twice. Is this what you're asking?

**Comment by kirubarajan** [comment]
LGTM

**Review by VinaiRachakonda — APPROVED** [review]

**Pull request merged**